kingfisher_scrapy/spiders/mexico_aguascalientes_sesea_plataforma_digital_nacional.py

```diff
@@ -0,0 +1,46 @@
+import json
+
+import scrapy
+
+from kingfisher_scrapy.base_spiders import IndexSpider
+from kingfisher_scrapy.util import handle_http_error
+
+
+class MexicoAguascalientesSESEAPlataformaDigitalNacional(IndexSpider):
+    """
+    Domain
+      Secretaría Ejecutiva del Sistema Estatal Anticorrupción de Aguascalientes (SESEA) - Plataforma Digital Nacional
+    Bulk download documentation
+      https://plataformadigitalnacional.org/contrataciones
+    """
+    name = 'mexico_aguascalientes_sesea_plataforma_digital_nacional'
+
+    # BaseSpider
+    root_path = 'data.item'
+
+    # SimpleSpider
+    data_type = 'release'
+
+    # IndexSpider
+    limit = '/pagination/pageSize'
+    result_count_pointer = '/pagination/total'
+    start_page = 0
+    use_page = True
+
+    # Local
+    url = 'https://api.plataformadigitalnacional.org/s6/api/v1/search?supplier_id=SESEA_AGS'
+
+    def start_requests(self):
+        yield scrapy.Request(self.url,
+                             meta={'file_name': 'page-0.json'},
+                             callback=self.parse_list,
+                             method='POST')
```

Comment:
Or do this indentation style:

```
some_thing = (
    a,
    b,
    c,
)
```

but never:

```
some_really_long_thing = (a,
                          b,
                          c)
```

```suggestion
        yield scrapy.Request(self.url, meta={'file_name': 'page-0.json'}, callback=self.parse_list, method='POST')
```

Reply:
Sorry, my IDE likes the second format!

Reply:
Ah, maybe it can be configured to use Black? https://marcobelo.medium.com/setting-up-python-black-on-visual-studio-code-5318eba4cd00 We haven't switched Kingfisher Collect to Black. Maybe there is another way to override the code style format.

Reply:
I will check that, thanks!